fili-core/src/main/java/com/yahoo/bard/webservice/exception/ExceptionHandler.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.exception;
+
+import com.yahoo.bard.webservice.web.apirequest.DataApiRequest;
+
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+import java.util.Optional;
+
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.core.UriInfo;
+
+/**
+ * Allows a customer to inject custom code for handling Exceptions in Fili.
+ *
+ * For example, customers may want to return 500's for some custom exceptions, or return
+ * an empty set when dimension rows are not found.
+ */
+public interface ExceptionHandler {
```

Comment:
Where is this interface used?

Reply:
This is vestigial, and should be removed.